analyzer/consensus/consensus.go

```diff
@@ -413,6 +413,11 @@ func (m *Main) queueTransactionInserts(batch *storage.QueryBatch, data *storage.
 			signedTx.Signature.PublicKey,
 		).String()
 
+		bodyBytes, err := tx.Body.MarshalCBOR()
```

Comment:
Is this because v5 doesn't use `MarshalBytes` anymore, and instead marshals to a pgx.Value?

Also, on the api side does this need to be unmarshalled as cbor?